### DISCUSSION TITLE
Measure per-NUMA memory bandwidth usage of a single core

### DISCUSSION THREAD

**Discussion by QiongwenXu** [opcm/pcm#776] [Q&A]
Hi, is it possible to measure per-NUMA memory (ie, dram) bandwidth usage of per core? For example, core 1 uses 1GB/sec memory bandwidth on NUMA 0, 2GB/sec memory bandwidth on NUMA 1, and 3GB/sec memory bandwidth on NUMA 2. Thanks!

**Comment by rdementi** [comment]
As of now I am not aware of such capability. One can only distinguish between local memory traffic and remote memory traffic for a core.
